### PR TITLE
fifottl: accelerate fifottl:take

### PR DESCRIPTION
**Pull request by olegrok**
* delete iteration over status index
* apply iteration with :pairs() over status index

### CODE REVIEW

**Comment by Totktonada** [comment]
Changes I made (force-pushed):

* Rebased on top of current master.
* Added `Fixes #63`.

**Comment by Totktonada** [comment]
Changes I made (force-pushed):

* Removed GE iterator and so status check inside the loop.

**Comment by Totktonada** [comment]
@RunsFor Can you please check this fix on some more real cases? I mean that it does not break anything.

File: queue/abstract/driver/fifottl.lua [hunk 1]
```diff
@@ -262,18 +262,17 @@ end
 function method.take(self)
     local offset = 0
```
Comment (rybakit) [inline queue/abstract/driver/fifottl.lua]: Seems like `offset` is not needed anymore?

Comment (Totktonada) [reply queue/abstract/driver/fifottl.lua]: Sure, thanks! Force-pushed.

**Comment by olegrok** [comment]
@Totktonada 

I checked it for our project. I suppose that this changes don't break anything. Let's merge it.

**Comment by Totktonada** [comment]
Thanks! Merged.

**Pull request merged**